packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts

```diff
@@ -78,10 +78,11 @@ export class SketchContainerComponent implements OnInit {
     try {
       const data = await this.service.process(file);
       this.store.dispatch(new CurrentFile(data));
-    } catch (e) {
+    } catch (error) {
+      console.error(error);
       this.store.dispatch(
         new InformUser(
-          'The design was created using an unsupported version of SketchApp.'
+          `Runtime Error: ${error}`
```

Comment:
I agree that when this happens it should not be a message as previous.
But maybe we can be a bit more helpful when this occurs. Something like: `If this already happend 
 multiple times please help us and make a issue`

What do you think?

Reply:
Yep. We can append a report link thank points to https://github.com/xlayers/xlayers/issues/new?assignees=&labels=&template=bug_report.md&title=